tests/ui/doc_errors.rs

```diff
@@ -1,3 +1,4 @@
+// compile-flags: --edition 2018
```

Comment:
Should I make a seperate file to keep the 2015 checks going?

Reply:
I think we don't need :)